src/hotspot/cpu/ppc/assembler_ppc.inline.hpp

```diff
@@ -168,6 +200,20 @@ inline void Assembler::lis(  Register d, int si16)             { Assembler::addi
 inline void Assembler::addir(Register d, int si16, Register a) { Assembler::addi(d, a, si16); }
 inline void Assembler::subi( Register d, Register a, int si16) { Assembler::addi(d, a, -si16); }
 
+// Prefixed instructions, introduced by POWER10
+inline void Assembler::pli(  Register d, long si34)             { Assembler::paddi_r0ok( d, R0, si34, false); }
+inline void Assembler::pla(  Register d, long si34)             { Assembler::paddi_r0ok( d, R0, si34, true); }
+inline void Assembler::pla(  Register d, Register a, long si34) { Assembler::paddi( d, a, si34, false); }
+inline void Assembler::psubi(Register d, Register a, long si34) { Assembler::paddi( d, a, -si34, false); }
+
+inline void Assembler::pli_or_li(Register d, long si34) {
```

Comment:
Defined but not used.

Reply:
It is intended to show an example of an efficient set of macros/functions for a prefixed instruction.  Since a non-prefixed (32-bit) instruction is more efficient if the argument fits within the operand fields because of smaller I-cache usage and no need for the padding nop, it is convenient to have a macro/function that select better format.  I intended to show an example of recommended set of useful macros/functions when adding support for a prefixed instruction.  I'm planning to use this function in my next patch.  Is it better to move this to the next patch?

Reply:
Same reason as the next comment (paddi_or_addi), but necessity of this one may be weaker.

Reply:
Ok.  See my other comments about unused/untestable code.